instrumentation/micrometer/micrometer-1.5/javaagent/build.gradle.kts

```diff
@@ -18,3 +18,24 @@ dependencies {
 
   testImplementation(project(":instrumentation:micrometer:micrometer-1.5:testing"))
 }
+
+tasks {
+  val testBaseTimeUnit by registering(Test::class) {
+    filter {
+      includeTestsMatching("*TimerSecondsTest")
```

Comment:
JFYI I have been interested in changing the javaVersion type of tests from calling `test` to calling `check` - I think `check.dependsOn(otherTest)` is a bit more idiomatic than `test.dependsOn` and avoids this annoying non-trivial boiler plate every time.